classes/class-sensei-media-attachments.php

```diff
@@ -3,30 +3,56 @@
 if ( ! defined( 'ABSPATH' ) ) exit;
 
 class Sensei_Media_Attachments {
-	private $dir;
-	private $file;
+	/**
+	 * The single instance of self.
+	 *
+	 * @var    self
+	 * @access private
+	 * @static
+	 * @since  2.0.0
+	 */
+	private static $instance = null;
+
 	private $assets_dir;
 	private $assets_url;
 	private $token;
 
-	public function __construct( $file ) {
-		$this->file = $file;
-		$this->dir = dirname( $this->file );
-		$this->assets_dir = trailingslashit( $this->dir ) . 'assets';
-		$this->assets_url = esc_url( trailingslashit( plugins_url( '/assets/', $this->file ) ) );
+	/**
+	 * Sensei_Media_Attachments constructor.
+	 *
+	 * @param $file
```

Comment:
We're not passing a parameter here.

Reply:
Fixed in 05152052be7073909e32c574502049c246ae9e15